primer.md

```diff
@@ -289,6 +289,28 @@ context, for the purposes of this CloudEvent attribute those meanings are not
 relevant and therefore using `id` for some other purpose beyond uniqueness
 checking is out of scope of the specification and not recommended.
 
+### data
+
+The `data` attribute of a CloudEvent may carry the event data itself. Event
+consumers can use the `datacontenttype` attribute of a CloudEvent to determine
+what the type is, for example `application/json` or `image/png`. There are no
+restrictions on the type of data that may be included with a CloudEvent. Event
```

Comment:
might want to talk about encoding format instead of "type" - so it's not confused with the "type" attribute

Reply:
Done